content/intro-to-storybook/ember/en/data.md

```diff
@@ -71,189 +82,87 @@ export const reducer = (state, action) => {
   }
 };
 
-export default combineReducers({
-  reducer,
-});
+export default reducers;
 ```
 
-Then we'll update our `TaskList` to read data out of the store. First let's move our existing presentational version to `app/components/pure-task-list.hbs` and `app/components/pure-task-list.js` and wrap it with a container.
-
-In `app/components/PureTaskList.hbs`:
-
-```handlebars
-{{!--app/components/pure-task-list.hbs --}}
-{{#if @loading}}
- <LoadingRow />
- <LoadingRow />
- <LoadingRow/>
- <LoadingRow />
- <LoadingRow />
-{{else if this.tasksInOrder}}
-  {{#each this.tasksInOrder as |task|}}
-    <Task
-      @task={{task}}
-      @pin={{fn @pinTask task.id}}
-      @archive={{fn @archiveTask task.id}}
-    />
-  {{/each}}
-{{else}}
-  <div class="list-items">
-    <div class="wrapper-message">
-      <span class="icon-check" />
-      <div class="title-message">You have no tasks</div>
-      <div class="subtitle-message">Sit back and relax</div>
-    </div>
-  </div>
-{{/if}}
+And finally our store (watch the import):
+
+```js
+// app/store.js
+import { createStore } from 'tracked-redux';
+import reducers from './reducers';
+
+export const store = createStore(reducers);
 ```
 
-In `app/components/task-list.hbs`:
+First we’ll construct a simple Redux store that responds to actions that change
+the state of tasks, in a file called `reducers/index.js` (intentionally kept
+simple):
 
-```handlebars
+## Using a Route
 
-{{!!-- app/components/task-list.hbs --}}
-<div>
-  <PureTaskList
-    @tasks={{this.tasks}}
-    @pinTask={{this.pinTask}}
-    @archiveTask={{this.archiveTask}}
-  />
-</div>
+With that we are able to connect our route with the redux store. Thanks to
+tracked-redux we have no artificial properties put on the prototype of our
+objects, instead we can explicitely declare each field we want. Here we are
+going to use the route to load the data and a controller with actions to modify
+it:
 
+```js
+// app/tasks/route.js
+import Route from '@ember/routing/route';
+import { store } from '../store';
+
+export default class TasksRoute extends Route {
+  model() {
+    // thanks to tracked redux, store.getState()
+    // returns a tracked state
+    // whenever the state changes, these will be reflected in the template
+    return store.getState().tasks.filter(
+      (t) => t.state === 'TASK_INBOX' || t.state === 'TASK_PINNED'
+    );
+  }
+}
 ```
 
-And `app/components/task-list.js` :
+And the controller with modifications where we are going to use our earlier
+defined actions:
```

Comment:
@gossi this could be phrased a bit different. We could probably go with:

"Once the route is created, we can now move onto creating a controller with the necessary actions. Inside the `tasks` folder we've created add a new file called `controller.js` with the following:" What do you think.